NEWS.md

```diff
@@ -1,6 +1,9 @@
 
 # 1.1.0
 
+* Accept browser, HTTPS, or SSH URL in GitHub repo specification,
```

Comment:
What do you mean by "browser"?

Reply:
But how is that different from just a HTTPS url?

Reply:
Or any URL within the repo is supposed to work? Like this one works as well: `https://github.com/r-lib/remotes/pull/108`?